.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "xpdf-4.03"]
```

Comment:
Is it necessary to change this file? Looks like the same module, just different protocol for access. I would not commit this file.

Reply:
Yes, the Docker build script was unable to pull the repository over SSH.